libcontainer/userns/userns_fuzzer.go

```diff
@@ -2,14 +2,8 @@
 
 package userns
 
-import (
-	"strings"
-
-	"github.com/opencontainers/runc/libcontainer/user"
-)
-
 func FuzzUIDMap(data []byte) int {
-	uidmap, _ := user.ParseIDMap(strings.NewReader(string(data)))
+	uidmap = string(data)
```

Comment:
oh! missed a `:` here; let me fix that; also can get rid of the intermediate variable

Reply:
done 👍

Reply:
@thaJeztah I think branch also needs a rebase with master, cilium/ebpf has deprecated code causing CI to fail.

Reply:
Ah, thanks! for the ping I thought I rebased, but must have forgotten 😅 

Anyway, I rebased and pushed again